builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -2,6 +2,7 @@
 import groovy.json.JsonSlurper
```

Comment:
see if you can remove this usage

Reply:
Using JsonSlurperClassic in place of JsonSlurper